pkg/eventlog/event.go

```diff
@@ -1,20 +1,40 @@
 package eventlog
 
 import (
+	"encoding/json"
 	"time"
 )
 
+// Op holds an operation type on an event
+type Op string
+
+const (
+	// OpAdd represents an add event
+	OpAdd Op = "ADD"
+	// OpDel represents a delete event
+	OpDel Op = "DEL"
+	// OpDep represents a deprecate event
+	OpDep Op = "DEP"
+)
+
 // Event is entry of event log specifying demand for a module.
+// It implements json.Marshaler
 type Event struct {
 	// ID is identifier, also used as a pointer reference target.
 	ID string `json:"_id" bson:"_id,omitempty"`
 	// Op is the operation on the event log. Valid values are "ADD"
 	// TODO: add support for "DEL"
-	Op string `json:"op" bson:"op"`
+	Op EventOp `json:"op" bson:"op"`
```

Comment:
this seems to be fixed in other PR

Reply:
@michalpristas I change this to `EventOp` in here, and that should address https://github.com/gomods/athens/pull/152/files#r193490061 as well